packages/components/src/color-picker/index.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { debounce, noop, partial } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Alpha from './alpha';
+import Hue from './hue';
+import Inputs from './inputs';
+import Saturation from './saturation';
+import { colorToState, simpleCheckForValidColor } from './utils';
+
+export default class extends Component {
```

Comment:
This component should have name `ColorPicker` so it would be possible to find it in React dev tools.

```js
export default class ColorPicker extends Component {
```

Reply:
Fixed at d50deda